src/main/webapp/assets/feedStyle.css

```diff
@@ -44,10 +43,175 @@
   cursor: pointer;
 }
 
+#modal-background {
+  background: white;
+  background-color: rgba(0, 0, 0, 0.4);
+  display: none;
+  height: 100%;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+}
+
+#modal-create-post {
+  background: white;
+  border-radius: 15px;
+  bottom: 15%;
+  box-shadow: 0 1px 10px grey, 0 -1px 10px grey;
+  left: 25%;
+  max-height: 450px;
+  padding: 30px;
+  position: fixed;
+  width: 50%;
+  overflow: auto;
+  z-index: 1;
+  top: 15%;
+}
+
+#modal-close {
+  background: none;
+  border: none;
+  font-size: 1.25rem;
+  outline: none;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+}
+
+#modal-title {
+  margin-left: 12px;
+}
+
+input[type=text] {
+  border: none;
+  border-radius: 50px;
+  font-size: 1rem;
+  margin-bottom: 0;
+  margin-top: 0;
+  outline: none;
+}
+
+input[type=number] {
+  background: rgba(27, 24, 24, 0.1);
+  border: none;
+  border-radius: 50px;
+  font: 400 1rem Arial, sans-serif;
+  margin-bottom: 15px;
+  margin-top: 0;
+  outline: none;
+  padding: 12px 20px;
+  width: 40px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  padding-left: 10px;
+  padding-right: 0;
+}
+
+select {
+  border: none;
+  border-radius: 50px;
+  display: inline-block;
+  font: 400 1rem Arial, sans-serif;
+  margin: 0;
+  outline: none;
+  padding-bottom: 6px;
+  padding-left: 8px;
+  padding-right: 1px;
+  padding-top: 6px;
+  width: 50px;
+}
+
+#modal-description {
+  background: rgba(27, 24, 24, 0.1);
+  border: none;
+  border-radius: 25px;
+  box-sizing: border-box;
+  font: 400 1rem Arial, sans-serif;
+  font-size: 1rem;
+  margin-bottom: 0;
+  margin-top: 0;
+  outline: none;
+  padding: 12px 20px;
+  width: 100%;
+}
+
+#modal-time-text {
+  display: inline-block;
+  margin-bottom: 0;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 0;
```

Comment:
you could combine these four lines into `margin: 0 10px` if you want!

(`margin: top right bottom left`)
(`margin: top&bottom right&left`)

Reply:
Good point! Ill change it